.github/workflows/pypi-cd.yml

```diff
@@ -112,16 +110,16 @@ jobs:
 
             - name: Set up Python
               if: ${{ !contains(matrix.build.RUNNER, 'self-hosted') }}
-              uses: actions/setup-python@v4
+              uses: actions/setup-python@v5
               with:
                   python-version: "3.10"
 
             - name: Set up Python older versions for MacOS
               if: startsWith(matrix.build.NAMED_OS, 'darwin')
               run: |
-                brew update
-                brew upgrade || true
-                brew install python@3.8 python@3.9
+                  brew update
+                  brew upgrade || true
+                  brew install python@3.9
```

Comment:
> python@3.8 has been deprecated! It will be disabled on 2024-10-14.

Any idea why it was installed? Do we still need old python versions?

Reply:
Don't think so, but out of scope so I'll merge and we can take it outside